scss/bootstrap/overrides/style/_panel.scss

```diff
@@ -1,8 +1,9 @@
 .panel {
   @include box-shadow($crm-panels-box-shadow);
-
+  border-radius: $border-radius-small;
   border: none;
   margin-bottom: $grid-gutter-width;
+  overflow: hidden;
```

Comment:
This overflow property required to git border-radius working...not sure if there is any better approach